### PR TITLE
Map Chunk loading, Chunks and Chunk Generation

### PR DESCRIPTION
**Pull request by i509VCB**
Maps some parts of the chunk loading and generation code, and clears up a few param names in some chunk related classes.
- method_20458 -> tryUnloadChunk
  - try

### CODE REVIEW

**Comment by i509VCB** [comment]
good lord clock makes hella commits

File: mappings/net/minecraft/server/world/ChunkTicketManager.mapping [hunk 1]
```diff
@@ -66,6 +67,8 @@ CLASS net/minecraft/class_3204 net/minecraft/server/world/ChunkTicketManager
 		ARG 2 pos
 		ARG 3 level
 		ARG 4 argument
+	METHOD method_21623 getTicket (J)Ljava/lang/String;
+		ARG 1 chunkPosLong
```
Comment (Runemoro) [inline mappings/net/minecraft/server/world/ChunkTicketManager.mapping]: `chunkPos` is enough, the type shouldn't be part of the name.

File: mappings/net/minecraft/server/world/ThreadedAnvilChunkStorage.mapping [hunk 2]
```diff
@@ -109,8 +110,16 @@ CLASS net/minecraft/class_3898 net/minecraft/server/world/ThreadedAnvilChunkStor
 	METHOD method_19488 getPointOfInterestStorage ()Lnet/minecraft/class_4153;
 	METHOD method_20441 releaseLightTicket (Lnet/minecraft/class_1923;)V
 		ARG 1 pos
+	METHOD method_20458 tryUnloadChunk (JLnet/minecraft/class_3193;)V
+		ARG 1 chunkPosLong
```
Comment (Runemoro) [inline mappings/net/minecraft/server/world/ThreadedAnvilChunkStorage.mapping]: Same here.

File: mappings/net/minecraft/world/chunk/ChunkStatus.mapping [hunk 3]
```diff
@@ -60,6 +60,7 @@ CLASS net/minecraft/class_2806 net/minecraft/world/chunk/ChunkStatus
 		ARG 5 task
 	METHOD method_20612 runNoGenTask (Lnet/minecraft/class_3218;Lnet/minecraft/class_3485;Lnet/minecraft/class_3227;Ljava/util/function/Function;Lnet/minecraft/class_2791;)Ljava/util/concurrent/CompletableFuture;
 	CLASS class_2807 Task
+		COMMENT
```
Comment (Runemoro) [inline mappings/net/minecraft/world/chunk/ChunkStatus.mapping]: Empty comment?

File: mappings/net/minecraft/world/chunk/ProtoChunk.mapping [hunk 4]
```diff
@@ -44,3 +48,4 @@ CLASS net/minecraft/class_2839 net/minecraft/world/chunk/ProtoChunk
 	METHOD method_16679 getSection (I)Lnet/minecraft/class_2826;
 		ARG 1 y
 	METHOD method_17032 setLightingProvider (Lnet/minecraft/class_3568;)V
+	METHOD method_22405 setBiomeArray (Lnet/minecraft/class_4548;)V
```
Comment (Runemoro) [inline mappings/net/minecraft/world/chunk/ProtoChunk.mapping]: `setBiomes`

**Comment by Runemoro** [comment]
Did you forget to push? You resolved my comments without any changes

File: mappings/net/minecraft/world/chunk/ChunkStatus.mapping (hunk 3 above)
Comment (liach) [inline mappings/net/minecraft/world/chunk/ChunkStatus.mapping]:
```suggestion
```

**Comment by i509VCB** [comment]
I had to go do something so I'll push in a few mins after one fix I have to do by hand

**Comment by i509VCB** [comment]
Issues have been resolved

**Review by liach — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**